jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/DiscoveryService.kt

```diff
@@ -22,44 +20,31 @@ public class DiscoveryService(
 	}
 
 	/**
-	 * Parses the given [input] and tries to fix common mistakes.
+	 * Parses the given [input] and tries to fix common mistakes. The returned candidates are in order
+	 * of most secure followed by most likely to work.
 	 *
-	 *   - Add protocol (https and http)
-	 *   - Add port (8096 for http and 8920 for https)
-	 *   - Add baseurl (/jellyfin)
-	 *
-	 * The returned candidates are in order of most secure followed by most likely to work.
 	 * See [AddressCandidateHelper] for more information.
 	 */
 	public fun getAddressCandidates(input: String): List<String> = AddressCandidateHelper(input).run {
 		addCommonCandidates()
-		prioritize()
 		getCandidates()
 	}
 
 	/**
 	 * Connects to the [servers] and returns a list of responding servers ordered by input order.
-	 * Uses multiple rules to determine the recommendation like:
-	 *
-	 *   - HTTPS above HTTP
-	 *   - Higher Jellyfin version
-	 *   - Respond time
-	 *
-	 * Also adds the addresses reported by the server into the mix when visiting the public server
-	 * information. Reported addresses are inserted before their parent server.
+	 * Uses the [RecommendedServerDiscovery] rules to determine the recommendation.
 	 *
 	 * If the input in [servers] is the output from [getAddressCandidates] the optimal server to use
-	 * will be the first returned server with a [RecommendedServerInfoScore] of GOOD.
+	 * will be the first returned server with a [RecommendedServerInfoScore] of GREAT or GOOD.
```

Comment:
Does that still apply?
Might be a bit misunderstandable regardless. Maybe swap it and write:
"the first server with a score of GREAT or GOOD will be the optimal server to use".

Also, KDoc link for `GREAT` and `GOOD` please.